tests/Apache.IoTDB.Tests/Apache.IoTDB.Tests.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Could u explain what we can do with this test csproj ? what's the expected behavior when we run this project?